Xamarin.Forms.Core/SelectedItemChangedEventArgs.cs

```diff
@@ -4,11 +4,22 @@ namespace Xamarin.Forms
 {
 	public class SelectedItemChangedEventArgs : EventArgs
 	{
+		[Obsolete("Please use the constructor that reports the items index")]
```

Comment:
We need to make these obsolete messages consistent with the rest of Forms. Something like
`[Obsolete("This constructor is obsolete as of version 3.5. Please use SelectedItemChangedEventArgs(object selectedItem, int selectedItemIndex) instead.")]`